app/ExposureNotificationContext.tsx

```diff
@@ -35,9 +35,12 @@ export type ExposureHistory = ExposureDatum[];
 interface ExposureNotificationsState {
   exposureHistory: ExposureHistory;
   hasBeenExposed: boolean;
+  userHasNewExposure: boolean;
   toggleHasExposure: () => void;
-  exposureNotificationAuthorizationStatus: ENAuthorizationStatus;
```

Comment:
ended up shortening these names as they were too long to read, no behavior changes were introduced around them.